build.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2020 Intel Corporation. All rights reserved.
+
+# stop on most errors
+set -e
+
+RELEASE_PLATFORMS=(byt cht bdw apl cnl icl tgl)
+PLATFORMS=()
+
+PATH=$pwd/local/bin:$PATH
+
+BIN_DIR=$(pwd)
+
+die()
+{
+  >&2 printf '%s ERROR: ' "$0"
+  # We want die() to be usable exactly like printf
+  # shellcheck disable=SC2059
+  >&2 printf "$@"
+  exit 1
+}
+
+print_usage()
+{
+    cat <<EOF
+Build FW and tplg from SOF repo and cp to this repo
+ENV XTENSA_TOOLS_ROOT is needed for FW XCC build
+usage: build.sh [options] platform(s)
+       -s SOF dir
+       -v release version
+       -m path to MEU tool
+       -a Build all platforms
+EOF
+}
+
+while getopts "s:v:m:a" OPTION; do
+  case "$OPTION" in
+    s) SOF_DIR=$OPTARG ;;
+    v) RELEASE_VERSION=$OPTARG ;;
+    m) MEU_TOOL_PATH=$OPTARG ;;
+    a) PLATFORMS=("${RELEASE_PLATFORMS[@]}") ;;
+    *) print_usage; exit 1 ;;
+  esac
+done
+shift $((OPTIND-1))
+
+# parse platform args
+for arg in "$@"; do
+  platform=none
+  for i in "${RELEASE_PLATFORMS[@]}"; do
+    if [ x"$i" = x"$arg" ]; then
+      PLATFORMS=("${PLATFORMS[@]}" "$i")
+      platform=$i
+      shift || true
+      break
+    fi
+  done
+  if [ "$platform" == "none" ]; then
+    echo "Error: Unknown platform specified: $arg"
+    echo "Supported platforms are: ${RELEASE_PLATFORMS[*]}"
+    exit 1
+  fi
+done
+
+# checkout branch
+git checkout -b "stable-$RELEASE_VERSION"
+
+# define folders
+SOF_FW_DIR="lib/firmware/intel/sof/$RELEASE_VERSION"
+PUBLIC_SIGNED_DIR="$SOF_FW_DIR/public-signed"
+INTEL_SIGNED_DIR="$SOF_FW_DIR/intel-signed"
+TPLG_DIR="lib/firmware/intel/sof-tplg-$RELEASE_VERSION"
+TOOLS_DIR="tools/$RELEASE_VERSION"
+
```

Comment:
@dbaluta you guys will need to update this after it's merged for your FW and topology layouts.